stdlib/Downloads.version

```diff
@@ -1,4 +1,4 @@
 DOWNLOADS_BRANCH = master
-DOWNLOADS_SHA1 = afd04be8aa94204c075c8aec83fca040ebb4ff98
+DOWNLOADS_SHA1 = 2aaaddbb34fda2349e862058a18246b10a16531d
```

Comment:
Reminder to self to revert this change and drop the Downloads.jl checksums if this is good: we'll merge that PR and update the version of the stdlib separately.